packages/frontend/src/pages/Tile/components/TableBanner/ImportCsvButton.tsx

```diff
@@ -115,7 +192,8 @@ export const ImportCsvModalContent = ({
     }
     setIsParsing(true)
     Papa.parse(file, {
-      transformHeader: (header) => header.trim(),
+      // transform empty headers to '(empty)'
+      transformHeader: (header) => header.trim() || '(empty)',
```

Comment:
nicee i was adding excel sheets that didn't start at the top in another PR and you handled it here, swee